pkg/collector/resourceutilization/accelerator/process_gpu_collector.go

```diff
@@ -78,8 +78,9 @@ func UpdateNodeGPUUtilizationMetrics(processStats map[uint64]*stats.ProcessStats
 				}
 				processStats[uintPid] = stats.NewProcessStats(uintPid, uint64(0), containerID, vmID, command)
 			}
-			processStats[uintPid].ResourceUsage[config.GPUSMUtilization].AddDeltaStat(utils.GenericSocketID, uint64(processUtilization.SmUtil))
-			processStats[uintPid].ResourceUsage[config.GPUMemUtilization].AddDeltaStat(utils.GenericSocketID, uint64(processUtilization.MemUtil))
+			gpuName := fmt.Sprintf("%s%v", utils.GenericGPUID, gpuID)
```

Comment:
@rootfs how did this solve the GPU MIG naming problem?
You are also using the GPU ID as the key as in the PR #1236.
Note that the `utils.GenericGPUID` will be the same for all MIG instance, so in the end, this is doing the same thing as in the PR #1236.

Also, note that the PR #1236 is fixing other things :)